server/src/repositories/person.repository.ts

```diff
@@ -248,18 +278,34 @@ export class PersonRepository implements IPersonRepository {
     return result;
   }
 
-  create(entity: Partial<PersonEntity>): Promise<PersonEntity> {
-    return this.personRepository.save(entity);
+  create(...entities: Partial<PersonEntity>[]): Promise<PersonEntity[]> {
+    return this.personRepository.save(entities);
   }
 
   async createFaces(entities: AssetFaceEntity[]): Promise<string[]> {
     const res = await this.assetFaceRepository.save(entities);
     return res.map((row) => row.id);
   }
 
-  async update(entity: Partial<PersonEntity>): Promise<PersonEntity> {
-    const { id } = await this.personRepository.save(entity);
-    return this.personRepository.findOneByOrFail({ id });
+  async upsertFaces(assetId: string, entities: AssetFaceEntity[], sourceType?: string): Promise<string[]> {
```

Comment:
I was confused by this. Upsert means "add or create", but this is actually deleting the existing faces and then replacing them with the new ones. If that is what it is actually doing, can we rename this to `replaceFaces` instead?

Reply:
Yeah, I can see how that can be confusing. I was thinking of it as upserting at the asset level, not the face level.  `replaceFaces` is less ambiguous about that.

Reply:
Fixed by 23179280e927563ea27b2918da0f9c7f15971090